envi/archs/i386/emu.py

```diff
@@ -2641,3 +2641,8 @@ def i_pextrw(self, op):
 
     def i_pextrd(self, op):
         self.i_pextrb(op, width=4)
+        
+    def i_vpcext(self, op):
+        # return value is stored in ebx
+        # if ebx == 0: Virtual PC detected, else: illegal instruction exception
+        pass
```

Comment:
might want to make ebx explicitly *not* 0 rather than relying on whatever is already in ebx?

Reply:
Thank you for inspiration, discussion, and your support! Highly appreciated!